tests/virtctl/imageupload.go

```diff
@@ -130,33 +131,34 @@ var _ = VirtctlDescribe("[sig-storage]ImageUpload", decorators.SigStorage, Seria
 		Entry("PVC", "pvc", validatePVC, libvmi.WithPersistentVolumeClaim),
 	)
 
-	DescribeTable("[test_id:11655]Create upload volume with force-bind flag should succeed", func(resource string, validateFn func(string)) {
-		sc, exists := libstorage.GetRWOFileSystemStorageClass()
-		if !exists || !libstorage.IsStorageClassBindingModeWaitForFirstConsumer(sc) {
-			Fail("Fail no wffc storage class available")
-		}
+	DescribeTable("[test_id:11655]Create upload volume with force-bind flag should succeed",
+		decorators.RequiresWFFCStorageClass, func(resource string, validateFn func(string)) {
+			sc, exists := libstorage.GetRWOFileSystemStorageClass()
+			if !exists || !libstorage.IsStorageClassBindingModeWaitForFirstConsumer(sc) {
+				Fail("Fail no wffc storage class available")
+			}
```

Comment:
is the tab added all over this file in the passed functor a must-have by the formatter rules? not sure it helps either readability or following the git history.

Reply:
yes unfortunately.. because of the lint that the test name is too long it has to add a new line which causes the added tab to the rest of the test..